src/js/module/component/button.js

```diff
@@ -36,4 +36,8 @@ export default View.extend({
         .html(view)
         .toggleClass('hidden', !this.model.get('visible'));
   },
+
+  proxyEvent(event) {
```

Comment:
Because DOM events are separated from Backbone.View instances, we need to proxy them somehow to Backbone Eventemitter. This way we can easily define events that we want to proxy with the same callback -  [more](http://backbonejs.org/#View-events)